sdk/js/src/examples/simpleSearcherSvm.ts

```diff
@@ -105,6 +105,17 @@ export class SimpleSearcherSvm {
     return decimals;
   }
 
+  generateComputeBudgetInstructions(): TransactionInstruction[] {
+    const computeLimitInstruction = ComputeBudgetProgram.setComputeUnitLimit({
+      units: 300000,
```

Comment:
maybe a comment explaining this, otherwise searchers may blindly use this value without measuring the CUs their txs take up

Reply:
for swap this value should almost always be the same and we can set it to be a reasonable value

Reply:
this is used for Limo though